app/Metadata/Extractor.php

```diff
@@ -341,14 +341,14 @@ public static function createFromFile(NativeLocalFile $file): self
 							$taken_at->setTimezone(new \DateTimeZone(date_default_timezone_get()));
 						}
 						// In the remaining cases the timezone information was
-						// extracted and the recording time is assumed exhibit
-						// to original timezone of the location where the video
-						// has been recorded.
-						// So we don't need to do anything.
+					// extracted and the recording time is assumed exhibit
+					// to original timezone of the location where the video
+					// has been recorded.
+					// So we don't need to do anything.
 						//
-						// The only known example are the mov files from Apple
-						// devices; the time zone will be formatted as "+01:00"
-						// so neither of the two conditions above should trigger.
+					// The only known example are the mov files from Apple
+					// devices; the time zone will be formatted as "+01:00"
+					// so neither of the two conditions above should trigger.
```

Comment:
Spurious and wrong change of indention in a file which hasn't been changed otherwise. The very first line (`// in the remaining cases`) and the empty line in the middle (`//  `) are still correct.

Please get control of your editor! This isn't the first time, but the second or third PR where this kind of stuff happens. This is annoying. There are also other cases like this in this PR which I didn't bother to comment on, because I did not feel like it. But here the change it not only annoying, it is even inconsistent.